build.gradle

```diff
@@ -196,6 +199,7 @@ test {
     dependsOn buildJniLib
     systemProperty 'tests.security.manager', 'false'
     systemProperty "java.library.path", "$rootDir/jni/release"
+    systemProperty "jdk.attach.allowAttachSelf", true
```

Comment:
this is required to mock static classes/methods

Reply:
Add a comment in the file itself.

Reply:
ack